src/node.h

```diff
@@ -517,7 +517,11 @@ typedef void (*promise_hook_func) (v8::PromiseHookType type,
                                    v8::Local<v8::Value> parent,
                                    void* arg);
 
-typedef double async_uid;
+typedef double async_id;
+typedef struct async_context {
```

Comment:
Fwiw you don’t need a typedef, just `struct async_context { ... };` will work fine.